core-services/ci-operator-configresolver/ci-operator-configresolver.yaml

```diff
@@ -1,14 +1,24 @@
-kind: List
-apiVersion: v1
-items:
+kind: Template
```

Comment:
Not sure I'm the biggest fan of this. Why not just record the route.host as the only diff?

Reply:
Because that entails:
* Adding a copy of the file
* Adding scripting and a job to verify the diff


With the template approach, it can be done within applyconfig. The goal here is to get rid of that again once migration is done.

Reply:
Don't we already have the diff and validator? I'm worried that these will be left as cruft after the migration

Reply:
yeah but the existing scripting is based on finding the files in specific folders in this is in a different one. WDYT about adding a "TODO (postmigration)" comment and opening an issue to resolve all postmigration TODOs once we migrated?

Reply:
I mean, we're in no rush to migrate before the 15th, Seems like we should do it right